llvm/test/Transforms/Coroutines/coro-split-musttail-armv6.ll

```diff
@@ -0,0 +1,60 @@
+; Test that ARMv6 correctly masks coroutine infrastructure from generating musttail calls
+; RUN: opt < %s -passes='cgscc(coro-split),simplifycfg,early-cse' -S \
+; RUN:     -mtriple=arm-none-eabi -mcpu=cortex-m0 | FileCheck %s
```

Comment:
It might be worth having an extra check line for an arch that does support tail calls, to show the difference between the two